tests/modules/fusioncatcher/fusioncatcher/test.yml

```diff
@@ -0,0 +1,7 @@
+- name: fusioncatcher fusioncatcher test_fusioncatcher
```

Comment:
You are missing an entry in `pytest_modules.yml`, currently the tests are not triggered.

Reply:
done